### DISCUSSION TITLE
tes for get a achivments

### DISCUSSION THREAD

**Discussion by coreymtaylor** [community/community#21120] [Other Feature Feedback, Questions, & Ideas] [answered]
tess

**Comment by coreymtaylor** [comment]
:v

**Reply by AlexandrWalker** [reply]
:v

**Comment by 0x1881** [comment]
:d

**Comment by MrSarno** [comment, marked as answer]
Hi there, @Latip176!

If you're curious about the Galaxy Brain achievement, I believe discussions created by the same person as the accepted answer do not count towards it. That is to say, to earn the achievement, you specifically need to be responding to questions asked by others.

Hope that helps! 🙂

**Reply by coreymtaylor** [reply]
Thanks you!

**Reply by coreymtaylor** [reply]
I think it work😅

**Comment by RumeysaaKopuz** [comment]
Thanks for answer
